metrics.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package cgroups;
+
+message Metrics {
+	map<string, HugetlbStat> hugetlb = 1;
+	PidsStat pids = 2;
+	CpuStat cpu = 3;
+	MemoryStat memory = 4;
+	BlkioStat blkio = 5;
+}
+
+message HugetlbStat {
+	uint64 usage = 1;
+	uint64 max = 2;
+	uint64 failcnt = 3;
+}
+
+message PidsStat {
+	uint64 current = 1;
+	uint64 limit = 2;
+}
+
+message CpuStat {
+	CpuUsage usage = 1;
+	Throttle throttling = 2;
+}
+
+message CpuUsage {
+	// values in nanoseconds
+	uint64 total = 1;
+	uint64 kernel = 2;
+	uint64 user = 3;
+	repeated uint64 per_cpu = 4;
+}
+
+message Throttle {
+	uint64 periods = 1;
+	uint64 throttled_periods = 2;
+	uint64 throttled_time = 3;
+}
+
+message MemoryStat {
+	uint64 cache = 1;
+	uint64 rss = 2;
+	uint64 rss_huge = 3;
+	uint64 mapped_file = 4;
+	uint64 dirty = 5;
+	uint64 writeback = 6;
+	uint64 pg_pg_in = 7;
+	uint64 pg_pg_out = 8;
+	uint64 pg_fault = 9;
+	uint64 pg_maj_fault = 10;
+	uint64 inactive_anon = 11;
+	uint64 active_anon = 12;
+	uint64 inactive_file = 13;
+	uint64 active_file = 14;
+	uint64 unevictable = 15;
+	uint64 hierarchical_memory_limit = 16;
+	uint64 hierarchical_swap_limit = 17;
+	uint64 total_cache = 18;
+	uint64 total_rss = 19;
```

Comment:
Use `gogo.customname` to remove the case flip (RSS -> Rss) above. Let me know if you need an example.

Reply:
I thought we had support in protobuild for this?  Because we removed all the `customname` tags in the proto files and it just does it for us now